tests/webgl/test_local_gemm.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, print_function
+
```

Comment:
write local test cases in normal testcase format so we can always use nose

Reply:
Changed.